src/tools/tests/recipe2plan-test.ts

```diff
@@ -43,51 +43,48 @@ $ tools/update-goldens \n\n`
     assert.lengthOf(decoded['particleSpecs'], 1);
   }));
   it('outputs a valid protocol buffer for resolved recipes', Flags.withDefaultReferenceMode(async () => {
-    const payload = await protoPayloadFor(`
-      particle Reader
-        data: reads Thing {name: Text}
-      particle Writer
-        data: writes Thing {name: Text}
+    assert.deepEqual(
+      await protoPayloadFor(`
+        particle Reader
+          data: reads Thing {name: Text}
+        particle Writer
+          data: writes Thing {name: Text}
 
-      @arcId('writeArcId')
-      recipe WritingRecipe
-        thing: create 'my-handle-id' @persistent
-        Writer
-          data: writes thing
+        @arcId('writeArcId')
+        recipe WritingRecipe
+          thing: create 'my-handle-id' @persistent
+          Writer
+            data: writes thing
 
-      @arcId('readArcId')
-      recipe ReadingRecipe
-        data: map 'my-handle-id'
-        Reader
-          data: reads data
-      
-      recipe ReadWriteRecipe
-        thing: create
-        Writer
-          data: writes thing
-        Reader
-          data: reads thing
-    `);
-
-    // Only validating that the output can be can be decoded as a ManifestProto and right counts.
-    // Tests for for encoding works are in manifest2proto-test.ts.
-    assert.deepEqual(payload, {
+        @arcId('readArcId')
+        recipe ReadingRecipe
+          data: map 'my-handle-id'
+          Reader
+            data: reads data
+        
+        recipe ReadWriteRecipe
+          thing: create
+          Writer
+            data: writes thing
+          Reader
+            data: reads thing
+      `), {
       particleSpecs: [{
-        name: 'Writer',
+        name: 'Reader',
```

Comment:
Interesting that the order changed...

Reply:
I've changed the logic that computes the list of particles, so it might have changed. Slightly weird, but ok.